package com.example. spinner;

import android.os. Bundle;
import android.widget.ArrayAdapter;
import android.widget.Spinner;
import android.widget.TextView;
import android. view. View;
import android.widget.AdapterView;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    Spinner spinner;

    TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super. onCreate (savedInstanceState);
        setContentView(R.layout.activity_main);

        spinner = findViewById(R.id.spinner);
        textView = findViewById(R.id.selected_text_view);

        String[] items = {"None","Java", "Python", "HTML", "C"};

        ArrayAdapter<String> adapter = new ArrayAdapter<>( this,
                android.R.layout.simple_spinner_dropdown_item, items);

        spinner.setAdapter(adapter);
        spinner. setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
            public void onItemSelected(AdapterView<?> parent, View view,
                                       int position, long id) {
                textView.setText("Selected: " + items[position]);
            }
            public void onNothingSelected(AdapterView<?> parent) {
            }
        });
    }
}